enrichment-web/src/main/java/eu/europeana/enrichment/web/controller/TranslationController.java

```diff
@@ -157,47 +140,11 @@
 			body.setItemId(itemId);

```

Comment:
this implementation doesn't allow the use of "all" keywords anymore

Reply:
The "all" functionality is still in place. This code is for the specific item of the story. With the `itemId = "all"` we capture th